daedalus/src/redemption_key_validation.js

```diff
@@ -115,94 +47,7 @@ function isValidPaperVendRedemptionKey(code) {
     }
 }
 
-function tests() {
```

Comment:
Tests have been removed as they can't be run within the Node now (as `atob` is not defined in the Node). 

I have checked with Nikola Glumac and he confirmed to me that `isValidRedemptionKey` and `isValidPaperVendRedemptionKey` will be used only in Electron render process where we have `atob` defined.

Previously we have shown that `isValidPaperVendRedemptionKey` works as expected.

We can show that `isValidRedemptionKey` works too by analysing previous implementation that was run https://github.com/menelaos/purescript-b64/blob/master/src/Data/String/Base64.purs#L120 : 
```
decode :: String -> Either Error String
decode str =
  if atobIsDefined
    then
      unsafeStringToUint8ArrayOfCharCodes <$> atob (toRfc4648 str)
        >>= decodeUtf8
    else
runFn3 _decodeNode Left Right (toRfc4648 str)
```

where in Electron render process the first branch of `if` statement was run:
```
unsafeStringToUint8ArrayOfCharCodes <$> atob (toRfc4648 str)
        >>= decodeUtf8
```

and if we remove `decodeUtf8` and `unsafeStringToUint8ArrayOfCharCodes` as we don't need them in our use case - we are left with only `atob (toRfc4648 str)` which translates to our js implementation. (we have previously verified and fixed `toRfc4648` and `atob` is the same as it has been FFI from js.